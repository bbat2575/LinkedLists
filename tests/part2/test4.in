NEW 1
1
INSERT 1 1 2
INSERT -1 2 3
INSERT 0 5 2
INSERT 0 -5 2
INSERT 0 1 {}
INSERT 0 1 {
INSERT 0 1 }
INSERT 0 a 2
INSERT a 0 2
INSERT a a 2
INSERT 0.0 0 2
INSERT 0. 0 2
INSERT .0 0 2
INSERT 0e0 0 2
INSERT 2e-6 0 2
INSERT 0 0.0 2
INSERT 0 0. 2
INSERT 0 .0 2
INSERT 0 0e0 2
INSERT 0 2e-6 2
INSERT  0 0 2
INSERT 0 0
INSERT 00
INSERT 0
INSERT
INSERT  
INSERT -0 0 2
INSERT +0 0 2
INSERT - 0 2
INSERT + 0 2
INSERT 0 - 2
INSERT 0 + 2
INSERT00
INSERT0
INSERT0 0
  INSERT
2 0 0 INSERT
 INSERT 0 0 2
  INSERT 0 0 2
INSER T 0 0 2
INS ERT 0 0 2
I NSERT 0 0 2
insert 0 0 2
Insert 0 0 2
INSErT 0 0 2
ISNERT 0 0 2
