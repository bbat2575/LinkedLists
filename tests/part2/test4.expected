
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
